.gitignore

```diff
@@ -5,6 +5,7 @@ frontend/node_modules/
 frontend/npm-debug.log
 frontend/yarn-error.log
 api/server/newUserController.js
+.github
```

Comment:
BTW `.github` shouldn't be ignored; otherwise we won't be able to modify issue templates and push them. It's just a directory for GitHub repo configs, and I've also put doc images inside it because I thought they were related